### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by HarryMichal**
Alongside the routine updates there's a major bump in the version of graphviz. Brief testing of GIMP (launch + minor editing of existing project) didn't show any potential regressions.

```
- cairo 1.17.4 -> 1.17.6
- graphviz 2.50.0 -> 3.0.0
- lcms 2.12 -> 2.13.1
- OpenBLAS 0.3.19 -> 0.3.20
- openexr 2.5.7 -> 2.5.8
- poppler 22.01.0 -> 22.03.0
- SuiteSparse 5.10.1 -> 5.11.0
```

### CODE REVIEW

File: org.gimp.GIMP.json [hunk 1]
```diff
@@ -691,22 +691,14 @@
...
                     "x-checker-data": {
                         "type": "anitya",
                         "project-id": 247,
                         "stable-only": true,
                         "url-template": "https://gitlab.freedesktop.org/cairo/cairo/-/archive/$version/cairo-$version.tar.gz"
                     }
-                },
-                {
-                    "type": "patch",
-                    "path": "patches/cairo-mr114-gimp-issue-6210.patch"
```
Comment (hfiguiere) [inline org.gimp.GIMP.json:705]: these two patches haven't been removed from the repository. They should be.

Comment (HarryMichal) [reply org.gimp.GIMP.json:705]: Good catch! Thanks!

**Comment by Jehan** [comment]
> Alongside the routine updates there's a major bump in the version of graphviz. Brief testing of GIMP (launch + minor editing of existing project) didn't show any potential regressions.

`graphviz` is only used for the `dot` CLI tool which comes with it, to draw graphs (in our case, the graph of the image processing tree). You can test if it's not broken like this:

* Start GIMP;
* Open or create an image;
* Go to menu `File > Edit > Show image graph`

Result: it should create a new image which will contain the drawing of your previously active image processing graph.

If it does, it's good. If it doesn't, it's not. ;-)

**Comment by HarryMichal** [comment]
Thanks @Jehan for the test steps! Just tried it and its working. Merging.

**Pull request merged**